docs/index.md

```diff
@@ -8,19 +8,95 @@ Welcome to the documentation of Parcels. **Parcels** provides a set of Python cl
 
-Here you'll find [installation instructions](installation.md), [tutorials and documentation](documentation/index.md), and the [API reference](reference.md) for Parcels. You can browse the documentation for older versions by using the version switcher in the left sidebar.
+**Version**: {{env.config.version}}
 
-If you need more help with Parcels, try the [Discussions page on GitHub](https://github.com/Parcels-code/parcels/discussions). If you think you found a bug, file an [Issue on GitHub](https://github.com/Parcels-code/parcels/issues). If you want to help improve Parcels, see the [Contributing](community/contributing.md) page.
+```{note}
+You can browse the documentation for older versions by using the version switcher in the bottom right.
+```
+
+**Useful links**: [Installation instructions](getting_started/installation.md) | [Discussions on GitHub](https://github.com/Parcels-code/parcels/discussions) | [Issue on GitHub](https://github.com/Parcels-code/parcels/issues) | [Parcels website](https://parcels-code.org/) | [CLAM community website](https://clam-community.github.io/) | [API reference](reference.md)
+
+`````{grid} 1 2 2 2
+:gutter: 4
+:padding: 2 2 0 0
+:class-container: sd-text-center
+
+````{grid-item-card} Getting started
+:shadow: md
+
+New to **Parcels**? Check out the installation guide, run the quickstart tutorial, and learn the key concepts to understand the package.
+
++++
+
+```{button-ref} getting_started/index
+:ref-type: doc
+:click-parent:
+:color: secondary
+:expand:
+
+Get started!
+```
+````
+````{grid-item-card} How to?
+:shadow: md
+
+Wondering how to load a `FieldSet` or write a `Kernel`? Find **tutorials** and explainers to these and other questions here:
+
++++
+
+```{button-ref} user_guide/index
+:ref-type: doc
+:click-parent:
+:color: secondary
+:expand:
+
+To the user guide
+```
+````
+````{grid-item-card} Contributing
+:shadow: md
+
+We encourage anyone to help improve **Parcels**: read our guidelines to get started!
+
++++
+
+```{button-ref} contributing/index
+:ref-type: doc
+:click-parent:
+:color: secondary
+:expand:
+
+Contributing guidelines
+```
+````
+````{grid-item-card} Community
+:shadow: md
+
+Want to interact with other users and **Parcels** developers?
+
++++
+
+```{button-ref} community/index
+:ref-type: doc
+:click-parent:
+:color: secondary
+:expand:
+
+Connect with our community!
+```
+````
+`````
 
 ```{toctree}
 :maxdepth: 2
 :hidden:
 
 Home <self>
 v4 <v4/index>
-Installation <installation>
-Tutorials & Documentation <documentation/index>
+Getting started <getting_started/index>
+Documentation <user_guide/index>
+Community <community/index>
+Contributing <contributing/index>
 API reference <reference>
-Contributing, Release Notes and more <community/index>
 Parcels website <https://parcels-code.org/>
```

Comment:
What's the plan for the "Getting started" section? Will there be a basic tutorial in there at some point?

I don't see a clear distinction in responsibilities between "Getting started" and "Documentation" currently - though I know that this is a WIP so maybe that's coming.

Also, thoughts on renaming `Documentation` to `User Guide` in the header?